utils/index.d.ts

```diff
@@ -1496,13 +1503,33 @@ export declare abstract class AzureWizardPromptStep<T extends IActionContext> {
      */
     public configureBeforePrompt?(wizardContext: T): void | Promise<void>;
 
+    /**
+     * Can be optionally added to a prompt step so the step info can be populated in the confirmation web view
+     */
+    public confirmationViewProperty?(wizardContext: T): ConfirmationViewProperty;
+
     /**
      * Return true if this step should prompt based on the current state of the wizardContext
      * Used to prevent duplicate prompts from sub wizards, unnecessary prompts for values that had a default, and to accurately describe the number of steps
      */
     public abstract shouldPrompt(wizardContext: T): boolean;
 }
 
+export type ConfirmationViewProperty = {
+    /**
+     *  A displayable name of the step
+     */
+    name: string;
+    /**
+     * A displayable value chosen by the user (The label of the chosen value for the step)
+     */
+    value: string;
+    /**
+     * The name which can be used to access the value in the wizard context
+     */
+    valueInContext: string;
```

Comment:
I think I'd prefer a name like `propertyName`, or `propertyKey`, or even just `key`. Could be more specific and do `contextPropertyName` or similar.